### ISSUE TITLE
React warnings in the browser console

### ISSUE THREAD

**Issue by shazron** [adobe-private/react-spectrum-v3#643]
# 🐛 Bug Report 

React warnings for:
1. React does not recognize the `focusedKey` prop on a DOM element
2. React does not recognize the `shouldCloseOnBlur` prop on a DOM element.

## 🤔 Expected Behavior

No warnings.

## 😯 Current Behavior

Warnings shown.

```
react-dom.development.js:88 Warning: React does not recognize the `focusedKey` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `focusedkey` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in div (created by ForwardRef($db588eb9c9dbc1dd892f18503cee21$var$ScrollView))
    in ForwardRef($db588eb9c9dbc1dd892f18503cee21$var$ScrollView) (created by ForwardRef($d62d682f3da39408e5c59b860cad8094$var$Virtualizer))
    in ForwardRef($d62d682f3da39408e5c59b860cad8094$var$Virtualizer) (created by ForwardRef($db61317aa22e4c1ceb7e68a142b3e28e$var$ListBoxBase))
    in ForwardRef($db61317aa22e4c1ceb7e68a142b3e28e$var$ListBoxBase) (created by ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker))
    in FocusScope (created by ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker))
    in div (created by ForwardRef)
    in ForwardRef (created by ForwardRef($af48fe067ffcbddd851563094a05$var$Popover))
    in Transition (created by $c2eed959492fcaa12e5ad56438b32db$export$OpenTransition)
    in $c2eed959492fcaa12e5ad56438b32db$export$OpenTransition (created by ForwardRef($efa130a588abe47e4415ae0ba92319d4$var$Overlay))
    in div (created by ForwardRef(ProviderWrapper))
    in ForwardRef(ProviderWrapper) (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ModalProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in I18nProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider) (created by ForwardRef($efa130a588abe47e4415ae0ba92319d4$var$Overlay))
    in ForwardRef($efa130a588abe47e4415ae0ba92319d4$var$Overlay) (created by ForwardRef($af48fe067ffcbddd851563094a05$var$Popover))
    in ForwardRef($af48fe067ffcbddd851563094a05$var$Popover) (created by ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker))
    in div (created by ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker))
    in ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker) (created by App)
    in ModalProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in I18nProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider) (created by ForwardRef($d618213cd07bac427c632ce5c90d7$var$Form))
    in form (created by ForwardRef($d618213cd07bac427c632ce5c90d7$var$Form))
    in ForwardRef($d618213cd07bac427c632ce5c90d7$var$Form) (created by App)
    in div (created by ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex))
    in ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex) (created by App)
    in div (created by ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex))
    in ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex) (created by App)
    in div (created by ForwardRef(ProviderWrapper))
    in ForwardRef(ProviderWrapper) (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ModalProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in I18nProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider) (created by App)
    in ErrorBoundary (created by App)
    in App
printWarning @ react-dom.development.js:88
react-dom.development.js:88 Warning: React does not recognize the `shouldCloseOnBlur` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `shouldcloseonblur` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in div (created by ForwardRef)
    in ForwardRef (created by ForwardRef($af48fe067ffcbddd851563094a05$var$Popover))
    in Transition (created by $c2eed959492fcaa12e5ad56438b32db$export$OpenTransition)
    in $c2eed959492fcaa12e5ad56438b32db$export$OpenTransition (created by ForwardRef($efa130a588abe47e4415ae0ba92319d4$var$Overlay))
    in div (created by ForwardRef(ProviderWrapper))
    in ForwardRef(ProviderWrapper) (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ModalProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in I18nProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider) (created by ForwardRef($efa130a588abe47e4415ae0ba92319d4$var$Overlay))
    in ForwardRef($efa130a588abe47e4415ae0ba92319d4$var$Overlay) (created by ForwardRef($af48fe067ffcbddd851563094a05$var$Popover))
    in ForwardRef($af48fe067ffcbddd851563094a05$var$Popover) (created by ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker))
    in div (created by ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker))
    in ForwardRef($f01f0d1066865f7027d77d32f5de6fe$var$Picker) (created by App)
    in ModalProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in I18nProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider) (created by ForwardRef($d618213cd07bac427c632ce5c90d7$var$Form))
    in form (created by ForwardRef($d618213cd07bac427c632ce5c90d7$var$Form))
    in ForwardRef($d618213cd07bac427c632ce5c90d7$var$Form) (created by App)
    in div (created by ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex))
    in ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex) (created by App)
    in div (created by ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex))
    in ForwardRef($f9fee2c1db386bde8f28b36a814d1eb2$var$Flex) (created by App)
    in div (created by ForwardRef(ProviderWrapper))
    in ForwardRef(ProviderWrapper) (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ModalProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in I18nProvider (created by ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider))
    in ForwardRef($bc3300334f45fd1ec62a173e70ad86$var$Provider) (created by App)
    in ErrorBoundary (created by App)
    in App
printWarning @ react-dom.development.js:88
```

## 🔦 Context

We're deploying a Project Firefly app to the Adobe Experience Cloud Shell.

## 💻 Code Sample

1. Create a `Project Firefly` app from https://console.adobe.io and attach an `Adobe I/O Runtime` namespace to your `Production` workspace.
2. Install Adobe I/O CLI @3.6.0: `npm install -g @adobe/aio-cli`
3. Run:
```
mkdir test && cd test
# choose your console project and accept all defaults
aio app init 
aio app deploy
```
4. Go to the `To view your deployed application in the Experience Cloud shell:` url in your browser
5. Invoke the `generic` action.
6. View the warnings in the browser's developer console.

## 🌍 Your Environment

```
  System:
    OS: macOS 10.15.5
    CPU: (4) x64 Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz
    Memory: 135.99 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.16.0 - ~/.nvm/versions/node/v12.16.0/bin/node
    Yarn: Not Found
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.0/bin/npm
  Virtualization:
    Docker: 19.03.8 - /usr/local/bin/docker
  npmGlobalPackages:
    @adobe/aio-cli: 3.6.0
  CLI plugins:
    @adobe/aio-cli 3.6.0 (core)
    @oclif/plugin-plugins 1.7.10 (core)
    @oclif/plugin-autocomplete 0.2.0 (core)
    @oclif/plugin-help 2.2.3 (core)
    @oclif/plugin-not-found 1.2.4 (core)
    @oclif/plugin-warn-if-update-available 1.7.0 (core)
    @adobe/aio-cli-plugin-config 2.2.1 (core)
    @adobe/aio-cli-plugin-app 3.0.1 (core)
    @adobe/aio-cli-plugin-console 3.1.0 (core)
    @adobe/aio-cli-plugin-certificate 0.2.0 (core)
    @adobe/aio-cli-plugin-auth 2.2.0 (core)
    @adobe/aio-cli-plugin-events 1.0.0 (core)
    @adobe/aio-cli-plugin-runtime 1.8.0 (core)
    @adobe/aio-cli-plugin-info 1.0.0 (core)
```

```
    "@react-spectrum/button": "^3.0.0-rc.2",
    "@react-spectrum/form": "^3.0.0-rc.2",
    "@react-spectrum/layout": "^3.0.0-alpha.1",
    "@react-spectrum/link": "^3.0.0-alpha.1",
    "@react-spectrum/picker": "^3.0.0-alpha.1",
    "@react-spectrum/progress": "^3.0.0-rc.2",
    "@react-spectrum/provider": "^3.0.0-rc.2",
    "@react-spectrum/text": "^3.0.0-rc.0",
    "@react-spectrum/textfield": "^3.0.0-rc.2",
    "@react-spectrum/theme-default": "^3.0.0-rc.2",
    "@react-spectrum/typography": "^3.0.0-alpha.1",
    "@adobe/aio-sdk": "^2.3.0",
    "node-fetch": "^2.6.0",
    "cloudevents-sdk": "^1.0.0",
    "uuid": "^8.0.0",
    "core-js": "^3.6.4",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-error-boundary": "^1.2.5",
    "regenerator-runtime": "^0.13.5",
    "@adobe/exc-app": "^0.2.14",
```

## 🧢 Your Company/Team

Adobe I/O DevX / CLI team

**Comment by dannify** [comment]
Thanks, we will be doing a pass of warnings to pick up the strays before we release

**Comment by shazron** [comment]
Should be fixed in #668
